x-pack/platform/plugins/shared/maps/public/classes/styles/vector/components/symbol/custom_icon_modal.test.tsx

```diff
@@ -17,6 +17,44 @@ jest.mock('../../../../../kibana_services', () => ({
     };
   },
 }));
+jest.mock('@kbn/mapbox-gl', () => {
```

Comment:
Thanks for adding these tests, much better than the old snapshot test.

Instead of mocking `maplibregl`, how about mocking `IconPreview` instead. `CustomIconModal` does not use `maplibregl` so it would be better to just mock the component that does instead and make this more focused on a single unit.